apps/mull-api/src/app/app.module.ts

```diff
@@ -29,8 +29,8 @@ import { UserModule } from './user';
       port: 3306,
       username: environment.db.username,
       password: environment.db.password,
-      database: 'mull',
-      synchronize: !environment.production,
+      database: environment.production ? 'mull-prod' : 'mull-dev',
+      synchronize: true,
```

Comment:
I am aware that this is not recommended to be `true` on prod, but I wanted to generate the tables. What do you guys think would be a good alternative?

Reply:
We definitely need a new process in order to maintain the schema on prod if we're not automatically synchronizing. Create #151 for it!